config/routes.rb

```diff
@@ -1,39 +1,25 @@
 Rails.application.routes.draw do
-  get "/", to: "home#index"
 
-  get "/merchants", to: "merchants#index"
-  get "/merchants/new", to: "merchants#new"
-  get "/merchants/:id", to: "merchants#show"
-  post "/merchants", to: "merchants#create"
-  get "/merchants/:id/edit", to: "merchants#edit"
-  patch "/merchants/:id", to: "merchants#update"
-  delete "/merchants/:id", to: "merchants#destroy"
+  resources :merchants
+  resources :reviews, only: [:edit, :update, :destroy]
+  resources :orders, only: [:new, :update]
+  resources :items do
```

Comment:
I think we want to restrict this so that it doesn't make paths for `new` and `create`, because those are nested under `merchants`

Reply:
update made